packages/fileimport-service/Dockerfile

```diff
@@ -1,97 +1,74 @@
 ARG NODE_ENV=production
 
-FROM node:18-bookworm-slim@sha256:408f8cbbb7b33a5bb94bdb8862795a94d2b64c2d516856824fd86c4a5594a443 as build-stage
+FROM mcr.microsoft.com/dotnet/sdk:8.0-noble AS dotnet-build-stage
+WORKDIR /app
+COPY packages/fileimport-service/ifc .
+RUN dotnet publish ifc-converter.csproj -c Release -o output/
 
-ARG NODE_ENV
-ENV NODE_ENV=${NODE_ENV}
+
+FROM mcr.microsoft.com/dotnet/runtime:8.0-noble AS runtime
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 
 WORKDIR /speckle-server
 
 # install tini
 ARG TINI_VERSION=v0.19.0
 ENV TINI_VERSION=${TINI_VERSION}
-ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini ./tini
-RUN chmod +x ./tini
-
-# install wait
-ARG WAIT_VERSION=2.8.0
-ENV WAIT_VERSION=${WAIT_VERSION}
-ADD https://github.com/ufoscout/docker-compose-wait/releases/download/${WAIT_VERSION}/wait ./wait
-RUN chmod +x ./wait
+ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini /usr/bin/tini
+RUN chmod +x /usr/bin/tini
+
+RUN apt-get update -y \
+    && DEBIAN_FRONTEND=noninteractive apt-get install -y \
+     --no-install-recommends \
+     curl=8.5.0-2ubuntu10.5 \
+    && curl -fsSL https://deb.nodesource.com/setup_18.x | bash - \
+    && DEBIAN_FRONTEND=noninteractive apt-get install -y \
+     --no-install-recommends \
+     nodejs=18.20.5-1nodesource1 \
+    && npm install -g yarn@1.22.22 \
```

Comment:
this is not ok, we're not using yarn 1

we're using corepack enabled yarn in the workspace root